src/components/clusterprovider/azure/azure.ts

```diff
@@ -47,9 +47,14 @@ export interface WaitResult {
     readonly stillWaiting?: boolean;
 }
 
+export interface Subscription {
```

Comment:
This interface was needed since AKS extension expects subscription ID for create cluster page.